examples/next-login/src/pages/home/signup.tsx

```diff
@@ -0,0 +1,163 @@
+import { Text, Stack, Alert } from '@ultraviolet/ui'
+import { Icon } from '@ultraviolet/icons'
+import {
+  Form,
+  TextInputFieldV2,
+  Submit,
+  useForm,
+  RadioGroupField,
+  DateField,
+  useWatch,
+} from '@ultraviolet/form'
+import { useState } from 'react'
+import styled from '@emotion/styled'
+import { EMAIL_REGEX, mockErrors } from '../../constants'
+
+type FormValues = {
+  email: string
+  password: string
+  gender: string
+  firstname: string
+  lastname: string
+  birthdate: Date
+}
+
+const StyledSignUpContainer = styled(Stack)`
+  margin: 5vh 25vw;
+  background: ${({ theme }) => theme.colors.secondary.background};
+  padding: ${({ theme }) => theme.space['4']};
+`
+
+const StyledInput = styled(TextInputFieldV2)`
+  padding: ${({ theme }) => theme.space['1.5']}
+    ${({ theme }) => theme.space['0']};
+  width: 100%;
+`
+type AlertProps = {
+  alert: string
+  values: {
+    email?: string | undefined
+    password?: string | undefined
+    gender?: string | undefined
+    firstname?: string | undefined
+    lastname?: string | undefined
+    birthdate?: Date | undefined
+  }
+  age: number
+}
+const SignUpAlert = ({ alert, values, age }: AlertProps) => {
+  if (alert === 'success') {
+    return (
+      <Alert sentiment="success" title="Account Created" closable>
+        Welcome {values.gender} {values.firstname} {values.lastname}, {age}
+        year{age > 1 ? 's' : ''} old. Your email adress is {values.email}.
+      </Alert>
+    )
+  }
+  if (alert === 'too young') {
+    return (
+      <Alert sentiment="danger" closable>
+        Must be 1+
+      </Alert>
+    )
+  }
+  return null
+}
+
+const SignUp = () => {
+  const methods = useForm({
+    mode: 'onChange',
+    defaultValues: {
+      email: '',
+      password: '',
+      gender: '',
+      firstname: '',
+      lastname: '',
+      birthdate: new Date(),
+    },
+  })
+  const values = useWatch(methods)
+  const [alertSubmit, setAlertSubmit] = useState('default')
+  const [age, setAge] = useState(0)
+
+  const handleSubmit = (val: FormValues) => {
+    let timeDiff = Math.abs(Date.now() - val.birthdate.getTime())
+    const computedAge = Math.floor(timeDiff / (1000 * 3600 * 24) / 365.25)
+    setAge(computedAge)
+    if (computedAge < 1) {
+      setAlertSubmit('too young')
+    } else {
+      setAlertSubmit('success')
```

Comment:
```suggestion
      setAlertSubmit(computedAge < 1 ? 'too young' : 'success')
```
No need to use condition if / else :)